pkg/block/validations.go

```diff
@@ -0,0 +1,29 @@
+package block
+
+import (
+	"context"
+	"errors"
+	"fmt"
+)
+
+func ValidateInterRegionStorage(ctx context.Context, adapter Adapter, storageNamespace string) error {
+	blockstoreMetadata, err := adapter.BlockstoreMetadata(ctx)
+	if errors.Is(err, ErrOperationNotSupported) || blockstoreMetadata.Region == nil {
```

Comment:
1. I think it's a bug to check for `blockstoreMetadata.Region == nil`
In which situations are we expected to get a nil value?
It seems to me we either get an err or the Region field should be populated
2. Please add a debug log in this case

Reply:
The idea was to make `BlockstoreMetadata` more general.
So for example, in the future, it will also include other properties.
In this case, `BlockstoreMetadata` may return the other props and not `region`, which says it won't be an error, but just not implemented.

If you find this more confusing, I can remove this check.

Reply:
My question is, not whether Metadata stores other information regarding the blockstore but rather if there is any real life scenario where the Region property will be nil.
If it makes sense - leave this as it is.
If there's no current scenario where this can actually be valid - we should remove this